### DISCUSSION TITLE
Compatibility

### DISCUSSION THREAD

**Discussion by Ciobrelis** [RWTH-EBC/AixLib#1330] [General]
Is it compatible with OpenModelica? Although it is in the list of "Package manager" and one can install it, it does not load and gives an error. 

![image](https://user-images.githubusercontent.com/34867742/185605724-10b06a23-1f23-4262-b614-583811e02a40.png)

Tried load / unloading, but it does not help.

**Comment by Maghnie** [comment]
Hi @Ciobrelis, maybe the latest [pre-release](https://github.com/RWTH-EBC/AixLib/releases/tag/v1.3.1) solves this issue?
